test/integration/MathFloor/math floor.cs

```diff
@@ -0,0 +1,5 @@
+-
+using System;
+
+Math.Floor(foo);
+-
```

Comment:
Notice how GitHub is showing a red symbol at the end here. Each file should have a blank line to end.
![image](https://cloud.githubusercontent.com/assets/3335181/22911479/cb81083e-f214-11e6-94d8-89f24165f3c4.png)

http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline